### PR TITLE
Doc review

### PR DESCRIPTION
**Pull request by CorBott**
Very minor changes on a very nice piece of doc!
I suggest you also move the glossary up one level to make it more visible.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ You're welcome to come and procrastinate with us :)
 Instructions for contribution
 -----------------------------
 
-Create your dev database
+Create your dev database
```
Comment (pmourlanne) [inline CONTRIBUTING.rst]:
```suggestion
Create your dev database
```

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -15,7 +15,7 @@ Create your dev database
     $ export PGDATABASE=procrastinate PGHOST=localhost PGUSER=postgres
     $ createdb
 
-Installation for development
+Set up your development environment
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (pmourlanne) [inline CONTRIBUTING.rst]: The `^` below need to be as long as the title :o The CI is failing because of it

Comment (pmourlanne) [reply CONTRIBUTING.rst]: (It's the case in multiple locations, I let you look into it :))

File: docs/quickstart.rst [hunk 3]
```diff
@@ -86,10 +85,10 @@ At this point, nothing is running yet. We've just created a task, which is the t
 (or blueprint) for a job.
 
 Our task doesn't really have an impact on the world (a side effect). It doesn't write a
-file, or update a database, it doesn't make an API call. In the real life, this is a
+file, or update a database, it doesn't make an API call. In real life, this is a
 problem, because at this point, all the job is doing is wasting CPU cycle. In our case,
 though, we'll just monitor the standard output to see if our task executed successfully,
-we should be able to see the result of the "print" call
+and check the result of the "print" call
```
Comment (pmourlanne) [inline docs/quickstart.rst]: This changes the meaning of the sentence :o The goal is to make sure a result is displayed. We are not checking the result in itself, but rather the presence of something in the standard output.

File: docs/quickstart.rst [hunk 4]
```diff
@@ -123,7 +122,7 @@ Run a worker
 
     (venv) $ procrastinate --verbose --app=tutorial.app worker
 
-In the logs, you should see the values as they are computed.
+In the logs, you can see the values as they are computed.
```
Comment (pmourlanne) [inline docs/quickstart.rst]: This is really optimistic :>

File: procrastinate/retry.py [hunk 5]
```diff
@@ -31,7 +31,7 @@ def get_schedule_in(self, attempts: int) -> Optional[float]:
         ----------
         attempts:
             The number of previous attempts for the current job. The first time
-            a job is ran, `attempts` will be 0.
+            a job is run, `attempts` will be 0.
```
Comment (pmourlanne) [inline procrastinate/retry.py]: :notes: And I raaaaan :notes:

Comment (CorBott) [reply procrastinate/retry.py]: run, ran, run, je suis formelle :)
I run
I ran
I have run

Comment (pmourlanne) [reply procrastinate/retry.py]: Je ne contredisais pas :> Je me rappelais les annees 80 :santa:

File: docs/quickstart.rst (hunk 3 above)
Comment (ewjoachim) [reply docs/quickstart.rst]: I've changed it a bit :)

**Comment by ewjoachim** [comment]
@pmourlanne ready when you are :)

**Pull request merged**